extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -183,7 +217,111 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this._dialogObject.content = [tab];
 	}
 
+	private createResouceButtonsContainer(): azdata.ListViewComponent {
+		const tags = this.getAllResourceTags(this._resourceTypes);
+		if (!tags.includes('All')) {
+			tags.splice(0, 0, 'All');
+		}
+		const items: azdata.ListViewOption[] = [];
+		tags.forEach((t: string) => {
+			items.push({
+				label: t,
+				id: t
+			});
+		});
+		const listView = this._view.modelBuilder.listView().withProps({
+			title: {
+				text: 'Categories',
```

Comment:
localize

Reply:
@aasimkhan30 , please only resolve the comment if you have fixed it and pushed new commits, or you have answered it. I noticed the same issue for the other comments as well.

Reply:
Sorry, I had fixed the commits locally but forgot to push.